src/clj/puppetlabs/services/jruby/jruby_metrics_core.clj

```diff
@@ -107,6 +119,8 @@
               :queue-limit-hit-count schema/Int
               :queue-limit-hit-rate schema/Num}}})
 
+(declare borrow-timers)
```

Comment:
I could, but it felt better placed down in the "Public" section as it is intended to be called from other namespaces that need to get the list of borrow timers. The forward declaration is here because the status summary methods in the "Private" section use it.

Could definitely move it up here, but that seems like it might run counter to the existing organization of the file:

  - Definitions
  - Private functions
  - Public functions

Reply:
And `borrow-timers` would be used in SERVER-2198 after this and https://github.com/puppetlabs/trapperkeeper-webserver-jetty9/pull/192 are merged? Just making sure I follow the reasoning behind that visibility level.

Reply:
Not quite. The forward declaration of `borrow-timers` is there purely to preserve the existing organization of the source file.

If puppetlabs/trapperkeeper-webserver-jetty9#192 is merged, SERVER-2198 becomes a one-line addition to the `track-return-instance!` function:

    (MDC/put "jruby.borrow-time" elapsed-time)

Reply:
Okay. I wouldn't mark borrow-timers as a public function unless we have an external consumer now or plan to in the immediate future.